test/support/rummager_helpers.rb

```diff
@@ -18,15 +18,22 @@ def stub_content_for_taxon(content_ids, results, filter_navigation_document_supe
       )
   end
 
-  def stub_most_popular_content_for_taxon(content_id, results, filter_navigation_document_supertype: 'guidance')
+  def stub_most_popular_content_for_taxon(content_id, results, filter_content_purpose_supergroup: 'guidance_and_regulation')
+    fields = %w(title
```

Comment:
This could be a seperate method,  it may even be able to service the same need in  `stub_most_recent_content_for_taxon`